server/athenian/api/precompute/discover_accounts.py

```diff
@@ -28,5 +38,39 @@ async def main(context: PrecomputeContext, args: argparse.Namespace) -> List[int
         if state.finished_date is None:
             log.warning("Skipped account %d because the progress is not 100%%", account)
             continue
-        result.append(account)
-    return result
+
+        discovered.append((account, precomputed))
+
+    if args.partition:
+        result: Dict[str, List[int]] = {"precomputed": [], "fresh": []}
+        for precomputed, group_accounts in groupby(
+            sorted(discovered, key=itemgetter(1)), key=itemgetter(1),
+        ):
+            result_field = "precomputed" if precomputed else "fresh"
+            result[result_field] = [account for account, _ in group_accounts]
+        return result
+    else:
+        return [account for account, _ in discovered]
+
+
+async def _get_accounts(sdb: Database) -> List[Tuple[int, bool]]:
+    """Return the existing account IDs, each with a precomputed flag."""
+    # already precomputed accounts have at least one repo set and all their
+    # repository sets are precomputed
+    if sdb.url.dialect == "sqlite":
+        and_aggr = func.min
+    else:
+        and_aggr = func.bool_and
+    precomputed_expr = func.coalesce(and_aggr(RepositorySet.precomputed), False)
```

Comment:
We consider some account precomputed if the "all" reposet is precomputed. The other reposets can only be subsets of the main one. So we should add the condition `.name == RepositorySet.ALL` and remove the aggregation.

Reply:
done